### PR TITLE
SB-814 removal of unused FilterAccessDecisionManager and making security config more readable

### PR DESCRIPTION
**Pull request by fuss86**

### CODE REVIEW

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java [hunk 1]
```diff
@@ -74,24 +74,24 @@ public void init(WebSecurity web)
     protected void configure(HttpSecurity http)
             throws Exception
     {
-        http.sessionManagement()
-            .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
-            .and()
+        http
+            .addFilterBefore(strongboxAuthenticationFilter(),
```
Comment (carlspring) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java]: Could you merge this line with the one above and re-align all the ones below? Thanks!

**Comment by carlspring** [comment]
Thanks!

**Comment by fuss86** [comment]
>Fixed the indentation

Hi @carlspring , the indention was intentionally implemented like that to emphasize the separation of configuration things in security :P

**Comment by carlspring** [comment]
Heh, sorry! I changed it last night when I merged the pull... Feel free to revert it, if you feel strongly about it.

**Pull request merged**